Source/NSMenuItemCell.m

```diff
@@ -736,7 +784,23 @@ - (void) drawKeyEquivalentWithFrame: (NSRect)cellFrame
    */
   else if (![[_menuView menu] _ownedByPopUp] || (_imageToDisplay == nil))
     {    
-      [self _drawText: [self _keyEquivalentString] inFrame: cellFrame];
+      if (_keyEquivalentFont != nil)
+        {
+          NSDictionary       *attrs;
+          NSAttributedString *aString;
+          
+          attrs = [NSDictionary dictionaryWithObject: _keyEquivalentFont
+                                              forKey: NSFontAttributeName];
```

Comment:
I am not sure, but shouldn't we also set the colour and respect the enabled/disabled state for this as well?

Reply:
You are right. Fixed.